doc/tutorials/ios-app.md

```diff
@@ -270,30 +277,95 @@ output will appear similar to the following:
 ```bash
 INFO: Found 1 target...
 Target //ios-app:ios-app up-to-date:
-  bazel-bin/ios-app/ios-app.ipa
-INFO: Elapsed time: 0.565s, Critical Path: 0.44s
+  bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa
+INFO: Elapsed time: 0.141s, Critical Path: 0.00s
 ```
 
 ### Find the build outputs
 
 The `.ipa` file and other outputs are located in the
-`$WORKSPACE/bazel-bin/ios-app` directory.
+`$WORKSPACE/bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa` directory.
+
+### Build the app in the simulator
+
+`rules_apple` supports running an app directly in the iOS Simulator.
+Replace `build` with `run` in the previous command to both build and
+run the application:
+
+```bash
+bazel run //ios-app:ios-app
+```
+
+Note: `--ios_simulator_device` and `--ios_simulator_version` control which
+version and device will be used when launching the app.
+
+### Generate an Xcode project
+
+There are a few community-provided solutions (such as [rules_xcodeproj](https://github.com/buildbuddy-io/rules_xcodeproj)
+and [Tulsi](https://github.com/bazelbuild/tulsi)) to help generating Xcode 
+projects. By doing so, you will be able to write, debug and test 
+iOS/macOS/watchOS/tvOS applications as if you were using the Xcode build system.
+
+Let's see how to do so with `rules_xcodeproj`.
+
+Open the `WORKSPACE` file again and add the following:
 
-### Run and debug the app in the simulator
+```starlark
+http_archive(
+    name = "com_github_buildbuddy_io_rules_xcodeproj",
+    sha256 = "c76958e21b7ea48a40fe4130e57911670281f23be93965573b4e90be47d779b4",
+    strip_prefix = "rules_xcodeproj-47d9510b64878ae1893d355cbf134d6a4b6eb715",
+    url = "https://github.com/buildbuddy-io/rules_xcodeproj/archive/47d9510b64878ae1893d355cbf134d6a4b6eb715.tar.gz",
```

Comment:
I assume there's a reason you're pinning to this commit instead of the official 0.7.1 release from a few hours before?

Reply:
There was an important change to how you can define `top_level_targets` (https://github.com/buildbuddy-io/rules_xcodeproj/pull/965), so I just decided to get ahead and include it here. We can definitely try to keep this up to date and use release versions instead of commits from now on.

Reply:
Yeah, let's use this commit for now, since it's a big change in how the rule is used. After this we should just use released versions.